pkg/gds/service_test.go

```diff
@@ -83,25 +85,19 @@ func (s *gdsTestSuite) ResetConfig() {
 }
 
 func (s *gdsTestSuite) SetupSuite() {
-	var err error
-	require := s.Require()
 	gin.SetMode(gin.TestMode)
 
-	// Create database paths to unpack fixtures to
-	s.dbPaths = make(map[fixtureType]string)
-	for _, ftype := range []fixtureType{empty, small, full} {
-		s.dbPaths[ftype], err = ioutil.TempDir("testdata", "db-*")
-		require.NoError(err)
-	}
-
 	// Load the reference fixtures into memory for use in testing
 	s.loadReferenceFixtures()
 
-	// Generate the databases if required (depends on the loaded reference fixtures)
-	if *update || !pathExists(fullDBFixturePath) || !pathExists(smallDBFixturePath) {
-		s.generateDB()
+	// Start the Trtl server if required
+	if s.stype == storeTrtl {
+		s.SetupTrtl()
```

Comment:
`SetupTrtl` has to be called before `generateDB` because we use the GDS storage interface `CreateVASP`, `UpdateCertReq` to generate the fixtures in the database, and Trtl has to be running for those functions to work.